benchmarkList.md

```diff
@@ -106,7 +106,11 @@ DRB179-thread-sensitivity-yes.cc                    |Y6	    | Conflicting writes
 DRB180-miniAMR-yes.c                                |Y6	    | Race by shared index variable                                       												| MiniAMR app               
 DRB181-SmithWaterman-yes.c                          |Y6	    | Race appears with larger data size                                  												| Smith-Waterman app
 DRB183-atomic3_yes.c                                |Y3     | Races because the write is not protected by atomic                                          | [Sequential Consistency for Data-race-free Programs](https://github.com/verified-software-lab/sc4drf/tree/main/experiments/civl/extra)
-
+DRB185-bar1_yes.c                                   |Y3     | wrong 2-thread flag barrier using busy-waits, race [inline]                                               |[Sequential Consistency for Data-race-free Programs](https://github.com/verified-software-lab/sc4drf/tree/main/experiments/civl/extra)
```

Comment:
what does [inline] mean here?  There are multiple appearances of [inline] in this pull request.  @LChenGit

Reply:
It is from the original repo. I have removed them.